couchdb/10-docker-default.ini

```diff
@@ -1,7 +1,7 @@
 ; couchdb/local.d/package.ini
 
 [fabric]
-request_timeout = infinity
+request_timeout = 31536000 ; 1 year
```

Comment:
When using `infinity`, I got an error similar to this one: https://github.com/apache/couchdb/issues/3789
```
[error] 2023-02-27T12:10:48.341702Z couchdb@couchdb-1.local <0.754.0> 08bd1a6cec req_err(2329098955) unknown_error : timeout_value
    [<<"fabric_util:get_shard/4 L149">>,<<"fabric:get_security/2 L220">>,<<"chttpd_auth_request:db_authorization_check/1 L109">>,<<"chttpd_auth_request:authorize_request/1 L19">>,<<"chttpd:handle_req_after_auth/2 L406">>,<<"chttpd:process_request/1 L388">>,<<"chttpd:handle_request_int/1 L325">>,<<"mochiweb_http:headers/6 L140">>]
[notice] 2023-02-27T12:10:48.341813Z couchdb@couchdb-1.local <0.754.0> 08bd1a6cec haproxy:5984 192.168.240.8 admin GET /_users 500 ok 1
```

Reply:
I wonder if it should be 1 hour to match our load balancer timeouts?

Reply:
I think that not all requests get to CouchDb through load balancer. On the other hand, I have no strong feelings about this timeout.

Reply:
I agree... I suspect this was set for things like the sentinel db watcher which could theoretically run for a year. But it's probably a bad idea for it to run for a year, and having a timeout is probably a good idea. Happy to leave this as is and discuss separately.